.storybook/components/Icons.js

```diff
@@ -177,15 +177,15 @@ const Icons = () => {
                     <Wrapper key={id}>
                       <IconWrapper>
                         <Icon
-                          id={id}
+                          aria-labelledby={id}
                           size={icon.size}
                           css={iconStyles(color)}
                         />
                       </IconWrapper>
-                      <Label htmlFor="id">
```

Comment:
Not only can a `<label>` not be associated with an SVG, but this would never have worked anyways because it's associated to an (nonexistent) element with `id="id"` 😅

Reply:
Whoops